src/heapdiff.cc

```diff
@@ -223,68 +224,73 @@ static void manageChange(changeset & changes, const HeapGraphNode * node, bool a
     return;
 }
 
-static Handle<Value> changesetToObject(changeset & changes)
+static Local<Value> changesetToObject(changeset & changes)
```

Comment:
Why rename this?  Makes pulling future changes in more difficult.

Reply:
Handle is non-existent in the new v8 headers.

Reply:
Could you setup a type alias at the top of the file to avoid most of these diffs?